framework/cluster/src/main/java/com/cloud/cluster/ClusterServiceServletImpl.java

```diff
@@ -162,6 +166,20 @@ private String executePostMethod(final CloseableHttpClient client, final HttpPos
         return result;
     }
 
+    protected InetAddress getBindAddressIfAvailable() {
+        String bindAddressStr = ServerPropertiesUtil.getProperty("bind.interface");
```

Comment:
`bind.interface` is a bit misleading, maybe `bind.address` ?

Reply:
@weizhouapache I'm not sure why it is named that way
https://github.com/apache/cloudstack/blob/main/client/conf/server.properties.in#L20
https://github.com/apache/cloudstack/blob/main/client/src/main/java/org/apache/cloudstack/ServerDaemon.java#L252

I didn't want to change the property name as it already exist

Reply:
I checked the javadoc

https://javadoc.jetty.org/jetty-12/org/eclipse/jetty/server/AbstractNetworkConnector.html#setHost(java.lang.String)

<img width="2690" height="460" alt="image" src="https://github.com/user-attachments/assets/9129d76d-7c91-4f0b-a960-d53ca46a8a99" />

so IP/hostname/null/0.0.0.0 are valid values. maybe we need to consider them ?

Reply:
IP/null/hostname should have been okay. I've added check for wildcards